packages/gatsby-transformer-code-examples/README.md

```diff
@@ -0,0 +1,56 @@
+# Gatsby Transformer Code Examples
+
+This plugin transforms file contents into CodeExample types in available in graphql.
+
+## Installation
+
+`npm install --save @commercetools-docs/gatsby-transformer-code-examples`
+
+## Usage
+
+As a prerequisite configure `gatsby-source-filesystem` plugin to point the directory of the code examples, for example `src/code-examples`.
+
+Example `gatsby-config.js` content:
+
+```js
+// In your gatsby-config.js
+module.exports = {
+  plugins: [
+    {
+      resolve: `gatsby-source-filesystem`,
+      options: {
+        name: `code-examples`,
+        path: `${__dirname}/src/code-examples`,
+      },
+    },
+    `@commercetools-docs/gatsby-transformer-code-examples`,
```

Comment:
Since the plugin is putting every node from the complete Graphql that has the matching media type and all the file contents are loaded into the client side bundle in our theme I would feel better having a path prefix configuration in the code examples transformer that filters out only the content of a specific folder. 

default can be `/` but we would put  `/src/code-examples/` into our theme's config.  That would also have the nice side effect that users would not have to explicitly put  the  `code-examples` part into the MDX although it's conventional anyways.

Reply:
You mean a plugin option? Yeah that makes sense. I guess we can have it as a default `src/code-examples`, then you just use the component as

```
<CodeExample file="foo.js" />
```

which resolves to `src/code-examples/foo.js`.

Is that what you had in mind?

Reply:
Should we look into this as well? @davifantasia @nkuehn

Reply:
it's implemented already now, jsut not reflected yet in the 

for now it's a hardwired convention, if need comes up we can add an option that defaults to the that convention later.